.github/workflows/release-please.yml

```diff
@@ -127,7 +129,6 @@ jobs:
           with:
             tag_name: ${{ needs.release-please.outputs.release_tag_name }}
             files: |
-              config/webhook/certificate.yaml
```

Comment:
This file doesn't exist anymore and hasn't been released for a while so I removed